django/template/smartif.py

```diff
@@ -61,8 +61,8 @@ def eval(self, context):
                 return func(context, self.first, self.second)
             except Exception:
                 # Templates shouldn't throw exceptions when rendering.  We are
-                # most likely to get exceptions for things like {% if foo in bar
-                # %} where 'bar' does not support 'in', so default to False
+                # most likely to get exceptions for things like {% if foo in
+                # bar %} where 'bar' does not support 'in', so default to False
```

Comment:
This is a good example of pre-existing code treated (and wrapped) as text. I've allowed that pattern to remain elsewhere where introduced by the autofixer